src/test/java/gov/healthit/chpl/aqa/stepDefinitions/UserRegistrationSteps.java

```diff
@@ -32,6 +45,7 @@ public void userRegistrationPage() {
     @Then("^\"([^\"]*)\", points to \"([^\"]*)\"$")
     public void userRegistrationHealthITForm(final String textName, final String urlLink) {
         WebDriver driver = getDriver();
+        //UserRegistrationPage.createNewAccount(driver).click();
```

Comment:
not right away but this comment can be removed as click event is moved to new step

Reply:
Sure!